--- a/app/Http/Controllers/IndexController.php
+++ b/app/Http/Controllers/IndexController.php
@@ -3,15 +3,14 @@
 namespace App\Http\Controllers;

 use Inertia\Inertia;
-
-// use Modules\Billing\Models\Product;
+use Modules\Billing\Models\Product;

 class IndexController extends Controller
 {
     public function __invoke()
     {
         return Inertia::render('Index', [
-            // 'products' => Product::displayable()->get(),
+            'products' => Product::displayable()->get(),
         ])->withSSR();
     }
 }
